package.json

```diff
@@ -27,10 +27,9 @@
   "license": "GPL-2.0",
   "description": "Connect allthethings",
   "devDependencies": {
-    "@slack/web-api": "^5.8.0",
+    "@slack/web-api": "^5.15.0",
```

Comment:
Several version got bumped to the latest compatible version (by `npm update`). I figured this was a good time while we we're tearing everything up anyway 🤷